Bottom: fef07f19d2b22e4845afe43c41ca7d1be2ab51ca
Top:    5d2bc1f3a0be8814734d927b684e5aac554c0dab
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

No need to delete /etc/mailname before overwrite


---
diff --git a/docker/docker-exim4/docker-entrypoint.sh b/docker/docker-exim4/docker-entrypoint.sh
index a99acea..7080bd4 100755
--- a/docker/docker-exim4/docker-entrypoint.sh
+++ b/docker/docker-exim4/docker-entrypoint.sh
@@ -25,8 +25,6 @@ case ${1} in
           && [[ ! -f ${CONFIG_PATH}/exim4.conf ]]
         then
             # set the hostname
-            [ -e /etc/mailname ] \
-                && rm -rf /etc/mailname
             echo "${HOSTNAME}" > /etc/mailname
             # set local IP addr
             LOCAL_IP_ADDR=$(getent hosts $HOSTNAME | cut -d ' ' -f 1)
